src/components/FeedbackContent.js

```diff
@@ -46,6 +47,15 @@ export const FeedbackContent = ({ title, message, icon, action }) => (
 );
 
 const styles = StyleSheet.create({
+  offCard: {
+    flex: 1,
+    justifyContent: 'center',
+    alignSelf: 'stretch',
+    marginTop: 16,
+    marginBottom: 45,
+    backgroundColor: COLORS.backgroundColor,
+    marginHorizontal: 16,
```

Comment:
Everything in these lines are duplicated from card, can you create a common "class" for those?

Reply:
Maybe do the opposite, the default would be the "offCard" and the parameter is to whether to display it as a card (which would only add the shadows and borders)

Reply:
Refactored.